app/controllers/comments_controller.rb

```diff
@@ -1,8 +1,13 @@
 class CommentsController < ApplicationController
+
   def create
-    comment = Comment.create(comment_params)
-    CommentMailer.email(comment).deliver_later unless comment.for_application?
-    redirect_to comment.team || comment.application
+    comment = Comment.new(comment_params)
+
+    if comment.save
+    redirect_to comment.application
```

Comment:
What do you know? I FORGOT to push the last changes with the corrected indentation. Good news bad news situation